adapters/repos/db/vector/compressionhelpers/compression.go

```diff
@@ -478,6 +485,7 @@ func RestoreHNSWPQMultiCompressor(
 	encoders []PQEncoder,
 	store *lsmkv.Store,
 	allocChecker memwatch.AllocChecker,
+	vectorName string,
```

Comment:
can we use `targetVector` naming? it will be then consistent with all the code base